bundle/View/ContentRenderer.php

```diff
@@ -77,7 +74,11 @@ public function renderEmbeddedContent(string $viewType, array $parameters = []):
             '_controller' => 'ng_content:embedAction',
         ];
 
-        $view = $this->viewBuilder->buildView($baseParameters + $parameters);
+        try {
+            $view = $this->viewBuilder->buildView($baseParameters + $parameters);
+        } catch (Exception $exception) {
+            return '';
```

Comment:
Could be good idea to write something to log in debug mode?

Reply:
Yes, added in https://github.com/netgen/ibexa-site-api/pull/12/commits/0f9a10cbf814aeb23762d65d072c9e5b2a32d023